<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdInfoWindow</class>
 <widget class="QMainWindow" name="QxrdInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_GridView">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="m_Splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="m_ObjectView"/>
      <widget class="QTreeView" name="m_PropertiesView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_FileMenu">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_EditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_WindowMenu">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionWindow"/>
   </widget>
   <addaction name="m_FileMenu"/>
   <addaction name="m_EditMenu"/>
   <addaction name="m_WindowMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWindow">
   <property name="text">
    <string>Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
